src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_DelegateCreationExpression.cs

```diff
@@ -26,15 +28,15 @@ public override BoundNode VisitDelegateCreationExpression(BoundDelegateCreationE
             if (node.Argument.Kind == BoundKind.MethodGroup)
             {
                 var mg = (BoundMethodGroup)node.Argument;
-                var method = node.MethodOpt;
+                var method = node.MethodOpt!;
```

Comment:
How do we know?

Reply:
Agreed. If we're sure, we should assert.